content/08.transmission.md

```diff
@@ -255,4 +256,30 @@ As a result, facets of systemic inequality such as mass incarceration of Black A
 
 #### Severity of COVID-19 Following Exposure
 
+Following exposure to SARS-CoV-2, the likelihood that an individual develops COVID-19 and the severity of the disease presentation can be influenced by a number of social factors.
+As discussed above, a number of patient characteristics are associated with the likelihood of severe COVID-19 symptoms.
+In some cases, these trends run counter to those expected given rates of exposure: for example, although women are more likely to be exposed, but men are more likely to be diagnosed with, hospitalized from, or die from COVID-19 [@doi:10.1371/journal.ppat.1008570].
+In the case of comorbid conditions and racial/ethnic demographics, however, social factors are highly likely to modulate or at least influence the apparent association between the patient trait and the increased risk from COVID-19.
```

Comment:
Do you mean patient here, or can that word be removed? I feel like this is true whether or not someone is seen in a clinical setting.

Reply:
Ah nope! I just got cognitively anchored on "patient traits" because we had such a long discussion coming up with that term!
```suggestion
In the case of comorbid conditions and racial/ethnic demographics, however, social factors are highly likely to modulate or at least influence the apparent association between these traits and the increased risk from COVID-19.
```